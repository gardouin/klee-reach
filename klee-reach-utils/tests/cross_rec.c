#include "klee/klee.h"
#define X 12

void bar();
void foo();

void foo(int x) {
	if (x > 0) {
		x--;
		bar(x);
	}
}

void bar(int x) {
	if (x > 0) {
		x -= 2;
		foo(x);
	}
}

int main() {
	foo(X);
	klee_reach();
	return 0;
}
