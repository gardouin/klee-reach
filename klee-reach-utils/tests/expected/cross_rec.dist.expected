56:0
24:1
20:10
53:10
19:11
18:12
44:12
17:13
43:13
16:14
42:14
41:15
40:16
13:2
21:2
12:3
48:3
11:4
37:4
45:4
10:5
36:5
35:6
9:6
34:7
8:7
33:8
55:8
32:9
54:9
